src/constants.js

```diff
@@ -0,0 +1,7 @@
+const API_KEY = "JuSHSq2nETP2FM9TtI0UKh23dL10s9M5";
```

Comment:
Use an `.env` file to store your api key and dont version that file. You can Provide a `.env.example` to express the usage of that var.

https://www.npmjs.com/package/dotenv